doc/awsrole.md

```diff
@@ -34,3 +34,28 @@ Options:
   - rolename: name of role to attach policy to
   - policyARN: arn of policy to attach to role
 
+### create-sa
+Create a new service account 
+```
+  gen3 awsrole create-sa <serviceaccount>
+```
+Options:
```

Comment:
This is all very cool, except lets get rid of `create-sa`.  I'd rather we have a `serviceaccount.yaml` file with each service (ex - `jenkins/serviceaccount.yaml`) that we just `kubectl apply -f .../fence/serviceaccount.yaml` ...  That will fit in nicely with HELM in the future too ...

It would actually be cool if we could manage the annotation in the `serviceaccount.yaml`, but not sure how to make that happen :-p